charts/otel-cloud-stack/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+name: otel-cloud-stack
+version: 0.1.0
+description: Helm chart for an Opentelemetry Kubernetes Quickstart
+type: application
+home: https://opentelemetry.io/
+sources:
+  - https://github.com/open-telemetry/opentelemetry-operator
+maintainers:
+  - name: jaronoff97
+  - name: anammedina21
+icon: https://raw.githubusercontent.com/cncf/artwork/a718fa97fffec1b9fd14147682e9e3ac0c8817cb/projects/opentelemetry/icon/color/opentelemetry-icon-color.png
+appVersion: 0.94.0
```

Comment:
This appversion corresponds with the collector, but the sources only list the operator.

Reply:
yeah i was thinking this would determine the version for everything this chart creates (read: collectors) but i also should pipe this in to the image used for the operator too